assets/pages/new-contributions-landing/components/SetPassword/SetPassword.jsx

```diff
@@ -0,0 +1,152 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { type Dispatch } from 'redux';
+
+import { type Contrib } from 'helpers/contributions';
+import { classNameWithModifiers } from 'helpers/utilities';
+
+import SvgPasswordKey from 'components/svgs/passwordKey';
+import SvgEnvelope from 'components/svgs/envelope';
+import CtaLink from 'components/ctaLink/ctaLink';
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+import { NewContributionTextInput } from '../ContributionTextInput';
+import { CreateAccountButton } from './CreateAccountButton';
+import { type ThankYouPageStage } from '../../contributionsLandingReducer';
+import { setThankYouPageStage, setPasswordHasBeenSubmitted, updatePassword, type Action } from '../../contributionsLandingActions';
+import { setPasswordGuest } from './helpers';
```

Comment:
should be an absolute path

Reply:
I'm not sure it should be?

Reply:
Good shout. Moved it to `helpers/setPassword`